docs/integrations/sources/amazon-ads-migrations.md

```diff
@@ -43,7 +43,7 @@ Depending on destination type you may not be prompted to reset your data.
 This will reset the data in your destination and initiate a fresh sync.
 ```
 
-For more information on resetting your data in Airbyte, see [this page](https://docs.airbyte.com/operator-guides/reset).
```

Comment:
Do we want to update the wording about resetting to clearing in general?

Reply:
yep.